README.md

```diff
@@ -148,6 +148,17 @@ return [
      */
     'x-service-checks' => [],
 
+    /**
+     * An array of cache stores can be provided for the Cache health
```

Comment:
Would be better if this matched the style of the other comments. Something like "A list of stores to be checked by the Cache health check"

Reply:
I prefer that too. Updating now.